quickstart/src/main/java/com/twilio/video/quickstart/activity/VideoActivity.java

```diff
@@ -291,6 +291,12 @@ protected  void onResume() {
          */
         encodingParameters = newEncodingParameters;
 
+        /*
+         * If audio routing switched to headset during a call, route through speaker.
+         */
+         AudioManager audioManager = (AudioManager) getSystemService(Context.AUDIO_SERVICE);
```

Comment:
This change would result in audio being routed through speaker every time the user returns to the app. We should restore the last set value instead. See https://github.com/twilio/video-quickstart-android/blob/master/quickstart/src/main/java/com/twilio/video/quickstart/activity/VideoActivity.java#L206.

Also add this change to `quickstartKotlin`